requirements.txt

```diff
@@ -2,7 +2,7 @@
 # Version updates managed by pyup.io
 
 attrdict==2.0.1
-broker>=0.1.3
+broker==0.1.8
```

Comment:
We've gone back and forth on this a bit, but using `>=` means pyup-bot won't ever update the minimum version for us automatically.

Since pip's default is not an eager update, I'm moving this back to pinned, and pyup-bot will automatically keep it up-to-date for us.